magda-search-api-node/src/createApiRouter.ts

```diff
@@ -162,15 +162,15 @@ export default function createApiRouter(options: ApiRouterOptions) {
 
             const processedQuery: Query = {
                 ...parseBaseQuery(queryString),
-                freeText: queryString.query
+                freeText: queryString.query as string
             };
 
             try {
                 const results = await searchQueryer.search(
                     processedQuery,
-                    queryString.start,
-                    queryString.limit,
-                    queryString.facetQuery
+                    queryString.start as any,
+                    queryString.limit as any,
+                    queryString.facetSize as any
```

Comment:
Probably worth to point out this change --- from the argument name it looks like a `facetSize` parameter. Thus, changed it from `facetQuery` to `facetSize`. Correct me if I were wrong

Reply:
👍